### PR TITLE
RFC: PUT endpoint for appending/modifying table data

### LINKED ISSUE #19: Write table append API RFC
See https://github.com/multinet-app/multinet-server/issues/483.

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #19

### CODE REVIEW

File: draft/XXXX-put-row-endpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+the `PUT` endpoint to append these tranches to reconstruct the whole table.
+
+## Proposals
+
+### `PUT` endpoint
+
+We are proposing to add a new endpoint:
+
+```
+PUT /api/workspaces/{workspace}/tables/{table}
```
Comment (jjnesbitt) [inline draft/XXXX-put-row-endpoint/README.md]: If the other endpoints (including your structure change section) use the `rows` suffix, shouldn't this as well?


```suggestion
PUT /api/workspaces/{workspace}/tables/{table}/rows
```

**Comment by jjnesbitt** [comment]
This looks good and concise!

File: draft/XXXX-put-row-endpoint/README.md (hunk 1 above)
Comment (waxlamp) [reply draft/XXXX-put-row-endpoint/README.md]: I'm usually a proponent of smaller changes done in sequence, but you are right, if we don't make your suggested change, we'll end up with an inconsistent API until we apply the fix in the structure section.

So what the hell, let's go ahead and make that change now.

**Review by JackWilb — APPROVED** [review]
LGTM. Thanks for this, Roni!

**Review by jjnesbitt — APPROVED** [review]

**Pull request merged**